plugin.video.hrmagplayer/libplayer/utils.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import urllib
+from libplayer.http import *
```

Comment:
This won't block the PR but generally it is a bad practice to do wildcard imports. Consider to only import what you need in future updates